recipe/meta.yaml

```diff
@@ -1,49 +1,52 @@
+{% set name = "holoviews" %}
 {% set version = "1.14.7" %}
 
 package:
-  name: holoviews
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/h/holoviews/holoviews-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/holoviews-{{ version }}.tar.gz
   sha256: 8d8d171227e9c9eaadd4b037b3ddaa01055a33bacbdbeb57a5efbd273986665f
 
 build:
-  number: 1
+  number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - holoviews = holoviews.util.command:main
 
 requirements:
   host:
-    - python >=2.7
+    - python
     - pip
     - pyct >=0.4.4
     - param >=1.7.0
     - setuptools >=30.3.0
     - wheel
   run:
     - python >=2.7
-    - param >=1.9.3,<2.0
-    - panel >=0.9.5
+    - colorcet
     - numpy >=1.0
+    - panel >=0.8.0
```

Comment:
Panel was pinned with `panel >=0.9.5` and here it is `panel >=0.8.0`. I'm not sure this older panel will necessarily work, but maybe that is a question for @philippjfr ?

Reply:
We can use `panel >=0.9.5`, but the upstream source uses `panel >=0.8.0` in setup.py, see https://github.com/holoviz/holoviews/blob/27da6a3af9e867e8e6432af3f7ceec19c854b332/setup.py#L17

Reply:
Ok, I see that master is different because it is the holoviews 2.0 branch and the 1.14 series is indeed using `panel >=0.8.0`.